docs/source/comparisons.rst

```diff
@@ -1,3 +1,34 @@
 ***********
 Comparisons
 ***********
+
+Forecast skill is always evaluated against a reference for verification. In ESM-based predictions, it is common to compare the ensemble mean forecast against the reference. This ensemble mean forecast (``comparison=e2r``) is expected to perform better than individual ensemble members (``comparison=m2r``) as the chaotic component of forecasts is expected to be suppressed by this averaging, while the memory of the system sustains. Todo: add citations to this
```

Comment:
Make sure to do ``comparison='e2r'`` and so on (with the single strings around to denote the str keyword)

Reply:
Also don't forget the Todo here. Might also be helpful to make more clear that this paragraph is in reference to HindcastEnsemble early on (sub-header?)

Reply:
>     1. I think the Murphy and probabilistic metrics docstrings/section should be filled out more. They are pretty bare right now in terms of what the metrics mean.

I still dont know too much about it. It contains the references towards the murcss page and reference to properscoring which explains it extensively.